pom.xml

```diff
@@ -188,6 +188,11 @@
             <version>2.3.1</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>no.digipost</groupId>
+            <artifactId>certificate-validator</artifactId>
+            <version>1.5</version>
+        </dependency>
```

Comment:
Denna är jag väldigt skeptisk till. Ska vara en väldigt god grund för att lägga till dependencies i embeddable klient-bibliotek. Är det bara pga den där `DigipostSecurity`? Isåfall tror jag vi får duplicera in här..